Swift Conversion

anuragdake opened the pull request:
Converted whole project to Swift 3.0.2.
Xcode used : 8.2.1

anuragdake committed:
Added icons

anuragdake committed:
Implemented IGCMenu.swift

anuragdake committed:
Added UI elements to Main.storyboard

anuragdake committed:
Added ViewController Code as example for IGCMenu
Updated IGCMenu.swift reference

anuragdake committed:
Added IBAction for Button in ViewController

anuragdake committed:
Added comments in IGCMenu and ViewController

anuragdake committed:
Rearranged code in IGCMenu.swift
Renamed few variables.

anuragdake committed:
Placed IGCMenu.swift at project level folder

sunilsharma08 reviewed (changes requested):
Thanks for converting to Swift.

There are some changes:
Make variable `pMenuButtonSuperView` optional,
If menu button size is zero then set default value not 0,
there is no need to initiate variable `menuItemsNameArray` if it is nil

anuragdake committed:
Made pMenuButtonSuperView optional
changed default value of menuButtonSize
delegate check to top when tap on menu button
